src/libraries/System.Runtime/tests/System.Runtime.Tests/System/TimeZoneInfoTests.cs

```diff
@@ -90,15 +90,25 @@ public static IEnumerable<object[]> Platform_TimeZoneNamesTestData()
                     { s_catamarcaTz, "(UTC-03:00) City of Buenos Aires", null, "Argentina Standard Time", "Argentina Daylight Time", "Argentina Summer Time" }
                 };
             else
-                return new TheoryData<TimeZoneInfo, string, string, string, string, string>
+            {
+                var testData = new TheoryData<TimeZoneInfo, string, string, string, string, string>
                 {
                     { TimeZoneInfo.FindSystemTimeZoneById(s_strPacific), "(UTC-08:00) Pacific Time (Los Angeles)", null, "Pacific Standard Time", "Pacific Daylight Time", "Pacific Summer Time"  },
-                    { TimeZoneInfo.FindSystemTimeZoneById(s_strSydney), "(UTC+10:00) Eastern Australia Time (Sydney)", null, "Australian Eastern Standard Time", "Australian Eastern Daylight Time", null },
                     { TimeZoneInfo.FindSystemTimeZoneById(s_strPerth), "(UTC+08:00) Australian Western Standard Time (Perth)", null, "Australian Western Standard Time", "Australian Western Daylight Time", null },
                     { TimeZoneInfo.FindSystemTimeZoneById(s_strIran), "(UTC+03:30) Iran Time", "(UTC+03:30) Iran Standard Time (Tehran)", "Iran Standard Time", "Iran Daylight Time", "Iran Summer Time" },
                     { s_NewfoundlandTz, "(UTC-03:30) Newfoundland Time (St. John’s)", null, "Newfoundland Standard Time", "Newfoundland Daylight Time", null },
                     { s_catamarcaTz, "(UTC-03:00) Argentina Standard Time (Catamarca)", null, "Argentina Standard Time", "Argentina Summer Time", null }
                 };
+
+                // Skip Sydney timezone test for Android due to inconsistent display name formatting
+                // Some Android devices return "(UTC+10:00) Australian Eastern Time (Sydney)" (see https://github.com/dotnet/runtime/issues/117903)
```

Comment:
Please don't skip it, instead validate against possible values.

Reply:
Only some devices received the update, meaning we have an uncoherent queue, so we would have to allow two correct answers for Android.

Reply:
I mean check against all display name values on Android. It is good to have the test explicitly listing those for clarity. Otherwise, we'll be hiding the behavior.

Reply:
Or are we getting empty name?

Reply:
Thanks a lot. There is actually an `alternativeDisplayName` which we could use for this. So we would, for Android, allow both "(UTC+10:00) Eastern Australia Time (Sydney)" and "(UTC+10:00) Australian Eastern Time (Sydney)".